.circleci/config.yml

```diff
@@ -23,6 +25,9 @@ jobs:
       # Build project
       - run: cp config.conf.example config.conf
       - run: make -j8 || true
+
+      # Run unit tests
+      - run: grunt qunit
```

Comment:
Move this up to with the `run tests` block.

Reply:
This has to be done after `make -j8` is called.

Reply:
Fair, then let's move the `make` stuff up.